src/main/resources/db/migration/V1.0.4__init_work_types.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS work_types;
+CREATE TABLE work_types
+(
+    id TEXT,
+    label TEXT,
+    PRIMARY KEY (id)
+);
+
+INSERT INTO work_types(ID, LABEL) VALUES
+    ('hearing-work','Hearing work'),
+    ('upper-tribunal','Upper Tribunal'),
+    ('routine-work','Routine work'),
+    ('decision-making-work','Decision-making work'),
+    ('applications','Applications'),
+    ('priority','Priority'),
+    ('access-requests','Access requests'),
+    ('error-management','Error management');
```

Comment:
Just thinking if these ought to be `camelCase` or `kebab-case` or `snake_case`.  I think it was put into the story without thinking about consistency elsewhere.

Reply:
I made the ids snake case